e2e-tests/README.md

```diff
@@ -2,14 +2,14 @@
 
 ### How to setup backstage configuration during the pipeline
 
-[configmap-app-config-rhdh.yaml](.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml) is the
+[configmap-app-config-rhdh.yaml](../.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml) is the
```

Comment:
now the name of the file is `app-config-rhdh.yaml` `.ibm/pipelines/resources/config_map/app-config-rhdh.yaml`